beacon_node/execution_layer/src/lib.rs

```diff
@@ -318,12 +321,17 @@ impl<T: EthSpec> ExecutionLayer<T> {
 
         let builder = builder_url
             .map(|url| {
-                let builder_client = BuilderHttpClient::new(url.clone()).map_err(Error::Builder);
-                info!(log,
+                let builder_client = BuilderHttpClient::new(url.clone(), builder_user_agent)
+                    .map_err(Error::Builder)?;
+
+                info!(
+                    log,
                     "Connected to external block builder";
                     "builder_url" => ?url,
-                    "builder_profit_threshold" => builder_profit_threshold);
-                builder_client
+                    "builder_profit_threshold" => builder_profit_threshold,
+                    "user_agent" => builder_client.get_user_agent(),
```

Comment:
```suggestion
                    "local_user_agent" => builder_client.get_user_agent(),
```

Nitpick, feel free to ignore if you like.